Week2/homework/maartjes-work.js

```diff
@@ -44,18 +44,29 @@ const tuesday = [
 
 const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
+const timeThreshold = 120;
+
+// function computeEarnings(tasks, hourlyRate) {
+//   return tasks
+//     .filter(task => task.duration >= timeThreshold)
+//     .reduce((acc, current) => acc + (current.duration / 60) * hourlyRate, 0);
```

Comment:
Wouldn't `task` be a better variable name than `current` here?

Reply:
Definitely yes. Changed to `task`